Author: mva
Description: Backport fix to call deduplication code by Vadim A. Misbakh-Soloviov.
 Upstream commit 70c319c79aee4c4829587cfa7a53a3e1518378f6
 
--- a/src/call-handler.cpp
+++ b/src/call-handler.cpp
@@ -83,7 +83,7 @@
             continue;
         }
         //check if any call manager is already handling this channel
-	if (handledCallChannels.contains(callChannel)) {
+	if (!handledCallChannels.contains(callChannel)) {
 	    handledCallChannels.append(callChannel);
 	    CallManager *manager = new CallManager(callChannel, this);
 	}
